proj/src/main/webapp/static/chat.html

```diff
@@ -173,7 +173,7 @@ <h1 class="display-4 card-header" id="big-title"> Chat Room </h1>
                         <div class="input-group-append">
                             <span class="input-group-text attach_btn"><i class="fas fa-paperclip"></i></span>
                         </div>
-                        <textarea id="message-input" class="form-control type_msg" placeholder="Type your message..."></textarea>
+                        <textarea id="message-input" class="form-control type_msg" id="message-input" placeholder="Type your message..."></textarea>
```

Comment:
This isn't needed because it already has the id at the start of the tag.

Reply:
oop didn't catch that, deleting the duplicate right now